### PR TITLE
refactor: Validate formatting in `nodes-base` (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
Changes only in `.prettierignore`, `nodes-base/.eslintrc` and `nodes-base/package.json`. Rest is formatting.

### CODE REVIEW

File: packages/nodes-base/credentials/HubspotAppToken.credentials.ts [hunk 1]
```diff
@@ -1,4 +1,9 @@
-import { IAuthenticateGeneric, ICredentialTestRequest, ICredentialType, INodeProperties } from 'n8n-workflow';
+import {
```
Comment (netroy) [inline packages/nodes-base/credentials/HubspotAppToken.credentials.ts:1]: this could be 
```suggestion
import type {
```

**Review by netroy — APPROVED** [review]
LGTM

File: packages/nodes-base/credentials/HubspotAppToken.credentials.ts (hunk 1 above)
Comment (ivov) [reply packages/nodes-base/credentials/HubspotAppToken.credentials.ts:1]: This happens everywhere in nodes-base. Let's tackle separately?

Comment (netroy) [reply packages/nodes-base/credentials/HubspotAppToken.credentials.ts:1]: in that case, please feel free to merge.

Comment (ivov) [reply packages/nodes-base/credentials/HubspotAppToken.credentials.ts:1]: Tracking [here](https://linear.app/n8n/issue/N8N-5688/use-type-imports-in-nodes-base).

**Comment by janober** [comment]
Got released with `n8n@0.204.0`

**Pull request merged**